README.adoc

```diff
@@ -222,20 +185,22 @@ Successfully built ee99661593eb
 Successfully tagged ol-runtime:latest
 ----
 
-Each step of the build has a unique ID, which is an ID of an intermediate image. For example, step 2
+Each step of the build has a unique ID, which represents the ID of an intermediate image. For example, step 2
 has the ID `dfef8073fe42`, and step 4 has the ID `ee99661593eb`, which is also the ID of the final image.
 During the first build of your image, Docker caches every new layer as a separate image and reuses them
-for future builds for steps that didn't change. For example, if you run the build again, Docker reuses
-the images that it cached for steps 2 - 4. However, you can tweak the `CMD` instruction and change the
-`defaultServer` server to the `testServer` server, and then Docker reuses the intermediate images for only steps 2 - 3.
+for future builds for layers that didn't change. For example, if you run the build again, Docker reuses
+the images that it cached for steps 2 - 4. However, if you make a change in your Dockerfile, Docker would
+need to rebuild the subsequent layer since it also changed.
```

Comment:
"...Docker **needs** to rebuild the subsequent layer since **this layer** also changed." I thought "it" was a little ambiguous here.

Reply:
I clarified the "it" part. As for the "would" bit, I thought this falls under 3rd type of conditionals: https://www.grammarly.com/blog/conditional-sentences/, where you describe a circumstance that would be different if something different happened in the past.